src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -889,6 +926,9 @@ public static void registerTags() {
         // @description
         // Returns whether the element contains a specified element, case insensitive.
         // Can use regular expression by prefixing the element with 'regex:'.
+        // @example
+        // Returns "true", as this regex matches the date.
+        // - narrate <element[It will happen on 3/4/2022.].contains_text[regex:(\d{1,2})\/(\d{1,2})\/(\d{4})]>

```

Comment:
This is painful, maybe a simpler example regex. Also don't use `{}` in examples, liable to have validation side effects.